Avoid implicit function declarations, for improved compatibility with
future compilers.

Fixed upstream via:

commit 562008027f343127d70bae44457cf8f06c666428
Author: Ralph Little <littlesincanada@yahoo.co.uk>
Date:   Thu Aug 29 21:49:08 2019 -0700

    m4/sane.m4: remove m4/sane.m4 and update aclocal.m4 to suit.
    
    We are now using pkg-config and it is pretty much a one-liner.
    No point in a whole file in m4 for just this feature.

diff -up xsane-0.999/m4/sane.m4.c99 xsane-0.999/m4/sane.m4
--- xsane-0.999/m4/sane.m4.c99	2023-01-17 17:11:08.940224113 +0100
+++ xsane-0.999/m4/sane.m4	2023-01-17 17:11:11.565198082 +0100
@@ -44,6 +44,7 @@ dnl
       AC_TRY_RUN([
 #include <sane/sane.h>
 #include <stdio.h>
+#include <stdlib.h>
 
 int 
 main ()
